staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -255,13 +255,13 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 	switch err.(type) {
 	case *restclient.RequestConstructionError:
 		// We will construct the request the same next time, so don't keep trying.
-		klog.Errorf("Unable to construct event '%#v': '%v' (will not retry!)", event, err)
+		klog.Errorf("Unable to construct event '%#v, Message: %s': '%v' (will not retry!)", event.InvolvedObject, event.Message, err)
 		return true
 	case *errors.StatusError:
 		if errors.IsAlreadyExists(err) {
-			klog.V(5).Infof("Server rejected event '%#v': '%v' (will not retry!)", event, err)
+			klog.V(5).Infof("Server rejected event '%#v, Message: %s': '%v' (will not retry!)", event.InvolvedObject, event.Message, err)
```

Comment:
on v(5) it seems very reasonable to include the entire message.

Reply:
yes， it reasonable.